src/routes/safe/components/Apps/utils.ts

```diff
@@ -147,3 +147,19 @@ const canLoadAppImage = (path: string, timeout = 10000) =>
       resolve(false)
     }
   })
+
+// Some apps still need chain name, as they didn't update to chainId based SDK versions
+// With naming changing in the config service some names aren't the expected ones
+// Ex: Ethereum -> MAINNET, Gnosis Chain -> XDAI
+export const getLegacyChainName = (chainName: string, chainId: string): string => {
+  let network = chainName
+  switch (chainId) {
+    case '1':
+      network = 'MAINNET'
+      break
+    case '100':
+      network = 'XDAI'
+  }
```

Comment:
Can it be an enum?
```
enum legacyNetworkNames {
  '1': 'MAINNET',
  '100': 'XDAI',
}
return legacyNetworkNames[chainId] || chainName
```

Reply:
It's adding complexity, because we can't use a number as `enum` key, so we should add more characters to the chainId in order to filter...

Reply:
It's a string though?

Reply:
Even you wrap it in '' typescript doesn't wan't them because it can cause confusion between the string id and the enum entry value

In the case above would cause something like this:
legacyNetworkNames[1] = 'XDAI'

And you expect
legacyNetworkNames['1'] = 'MAINNET'